stock_available_sale/tests/test_quoted_qty.py

```diff
@@ -5,60 +5,59 @@
 from odoo.tests import common
 
 
-class TestQuotedQty(common.SavepointCase):
+class TestQuotedQty(common.TransactionCase):
```

Comment:
Why did you change to TransactionCase? If there is not a good reason., leave it as it was please